### DISCUSSION TITLE
mastodon-streaming ERR! error: SSL required

### DISCUSSION THREAD

**Discussion by kfitz** [mastodon/mastodon#21397] [Q&A]
We've just migrated to a new server and are experiencing some odd problems. On the new server, we built out a full dev instance with its own domain info, and once the migration was complete we removed the dev DNS, connected the original production DNS, and set about rewriting all of the various config files to point to the production domain. Everything seems to be working, except for two things:

1. `systemctl status mastodon-streaming` shows repeated `ERR! error: SSL required`. We've checked nginx, reissued certificates, pored through every log we can find, and are finding nothing untoward except for that one repeated error. As a result, the home timeline isn't updating at all.
2. Additionally, doing just about anything in the web interface (posting, approving follows, etc) throws a 500 error. I'm guessing that's related to 1, but am not at all sure.

Any ideas would be MOST gratefully received!

**Comment by gasesjoyride** [comment]
Do you have an external database that needs SSL? I had this same error, and it turned out I needed to add `DB_SSLMODE=require` to `.env.production`
